### PR TITLE
fix: fill origin when creating or selecting a trackingNumber

### PR DESCRIPTION
**Pull request by gca-axelor**
RM#82116

### CODE REVIEW

File: packages/ui/src/components/molecules/EditableInput/EditableInput.tsx [hunk 1]
```diff
@@ -41,6 +41,10 @@ const EditableInput = ({
   const [isEditable, setEditable] = useState(true);
   const [value, setValue] = useState(defaultValue);
 
+  useEffect(() => {
+    setValue(defaultValue);
+  }, [defaultValue]);
```
Comment (vhu-axelor) [inline packages/ui/src/components/molecules/EditableInput/EditableInput.tsx:46]: I understand why you do that but I think it's better to do that in another ticket in dev

File: packages/apps/stock/src/screens/supplierArrivals/SupplierArrivalLineListScreen.js [hunk 2]
```diff
@@ -55,7 +55,7 @@ const SupplierArrivalLineListScreen = ({route, navigation}) => {
       lineDetailsScreen: 'SupplierArrivalLineDetailScreen',
       selectTrackingScreen: 'SupplierArrivalSelectTrackingScreen',
       selectProductScreen: 'SupplierArrivalSelectProductScreen',
-      skipTrackingNumberVerification: true,
+      skipTrackingNumberVerification: false,
```
Comment (vhu-axelor) [inline packages/apps/stock/src/screens/supplierArrivals/SupplierArrivalLineListScreen.js:58]: Revert this

**Review by vhu-axelor — COMMENTED** [review]
Add changelog please

File: packages/ui/src/components/molecules/EditableInput/EditableInput.tsx (hunk 1 above)
Comment (gca-axelor) [reply packages/ui/src/components/molecules/EditableInput/EditableInput.tsx:46]: i created a ticket, but i let it there so it works

File: packages/apps/stock/src/components/organisms/index.js [hunk 3]
```diff
@@ -17,6 +17,7 @@
  */
 
 export {default as DescriptionCard} from './DescriptionCard/DescriptionCard';
+export {default as EditableOriginInput} from './EditableOriginInput/EditableOriginInput';
```
Comment (vhu-axelor) [inline packages/apps/stock/src/components/organisms/index.js:20]: I don't think this component is an organism, it's more a molecule

File: packages/apps/stock/src/components/templates/customerDelivery/CustomerDeliveryLineTrackingNumberSelect/CustomerDeliveryLineTrackingNumberSelect.js [hunk 4]
```diff
@@ -45,8 +45,9 @@ const CustomerDeliveryLineTrackingNumberSelect = ({
 
   return (
     <ProductTrackingNumberSelect
+      trackingNumber={trackingNumber}
       product={product}
-      visible={visible}
+      visible={true}
```
Comment (vhu-axelor) [inline packages/apps/stock/src/components/templates/customerDelivery/CustomerDeliveryLineTrackingNumberSelect/CustomerDeliveryLineTrackingNumberSelect.js:50]: `visivble` prop is true by default so it's not useful here

File: changelogs/unreleased/82116.json [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{
+  "title": "Tracking Number: add possibility to fill origin when creating or selecting a trackingNumber",
```
Comment (vhu-axelor) [inline changelogs/unreleased/82116.json:2]: "Tracking number: add possibility to fill origin when creating or selecting"

**Review by vhu-axelor — APPROVED** [review]

**Pull request merged**